src/food.cpp

```diff
@@ -1,12 +1,46 @@
+#include <iostream>
+
 #include "food.hpp"
 
 
-Food::Food(const sf::Vector2f position) : _shape(size)
+Food::Food() : _type(FoodType::Neutral), _shape(sf::Vector2f(20.f, 20.f))
+{
+    _shape.setPosition(sf::Vector2f{});
+    _shape.setFillColor(sf::Color::White); 
+}
+
+Food::Food(const sf::Vector2f position, const sf::Vector2f size) : _type(FoodType::Neutral), _shape(size)
+{
+    _shape.setPosition(position);
+    _shape.setFillColor(sf::Color::White);
+}
+
+Food::Food(const sf::Vector2f position, const sf::Vector2f size, const FoodType type) :
+    _type(FoodType::Neutral), _shape(size)
 {
     _shape.setPosition(position);
-    _shape.setFillColor(_color);
+    _shape.setFillColor(sf::Color::White);
 }
```

Comment:
Too many constructors. Do you actually use them? Even if you do, I doubt it's necessary.

Reply:
Thanks. I think what third constructor is enougth